### ISSUE TITLE
Towncrier 19.9 compatibility

### ISSUE THREAD

**Issue by uranusjr** [sphinx-contrib/sphinxcontrib-towncrier#44]
Towncrier changed internals nad now the extension errors with

```
Extension error:
Handler <bound method TowncrierDraftEntriesEnvironmentCollector.get_outdated_docs of <sphinxcontrib.towncrier.TowncrierDraftEntriesEnvironmentCollector object at 0x10fb098b0>>
for event 'env-get-outdated' threw an exception
(exception: load_config_from_file() missing 1 required positional argument: 'config_file')
```

**Comment by webknjaz** [comment]
Thanks for the report!

@slsh1o do you want to try and fix this?

**Comment by webknjaz** [comment]
@Cadair I saw you mentioned this issue @ https://github.com/astropy/astropy/pull/11279#issuecomment-771161142. So FYI it's been fixed by @s0undt3ch in #47. I'll try to make a release soon.

Also TIL about https://github.com/OpenAstronomy/sphinx-changelog — I'll probably link it from the README pointing to similar projects. For now, just documenting that it exists in this comment tho.

**Comment by webknjaz** [comment]
Released as v0.2.0a0 at https://pypi.org/project/sphinxcontrib-towncrier/0.2.0a0/: https://github.com/sphinx-contrib/sphinxcontrib-towncrier/runs/2343058584?check_suite_focus=true#step:11:30
